ext/aom.cmd

```diff
@@ -8,12 +8,11 @@
 : # If you're running this on Windows, be sure you've already run this (from your VC2019 install dir):
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
 
-git clone -b v3.1.2 --depth 1 https://aomedia.googlesource.com/aom
+git clone -b v3.1.3 --depth 1 https://aomedia.googlesource.com/aom
```

Comment:
@junmer Could you please test the v3.1.3-rc1 tag on macOS with the Apple M1 chip? That tag is libaom v3.1.3 release candidate 1. Thank you for your help!

Reply:
the v3.1.3-rc1 on MacBook Pro (13-inch, M1, 2020) Apple M1 test passed.   thank you.

Reply:
Thank you for testing!

There is another file in libavif that contains the libaom version number. I don't remember it off the top of my head, but you can find it by searching for "v3.1.2" in the libavif source tree. Please also update that file in this pull request.